mkosi.images/base/mkosi.conf.d/10-debian-ubuntu.conf

```diff
@@ -22,7 +24,9 @@ Packages=
         libqrencode4
         libssl3
         libip4tc2
-        libtss2-dev # Use the -dev package to avoid churn in updating version numbers
+        libtss2-mu0
```

Comment:
this broke ABI again so it will stop working soon as the package had to change name

Reply:
Let's fix that when it breaks I say.

Reply:
that would be now, it's in unstable and will in a couple of days migrate to testing https://tracker.debian.org/pkg/tpm2-tss